src/components/ContinuationIn/AuthorizationProof.vue

```diff
@@ -111,9 +73,10 @@
       </v-row>
 
       <v-row
-        v-for="(num, index) in numFiles"
```

Comment:
This v-for syntax works because we were only looking at `index`.

I've updated this code to be more correct (and only de-reference the files array once).